chainlink-testing-framework/mask-toml-secrets/README.MD

```diff
@@ -0,0 +1,29 @@
+# Mask TOML Secrets Action
+
+This GitHub Action masks all secret values in Base64-encoded TOML configurations within GitHub Actions logs. It is designed to ensure that sensitive information denoted by keys ending with `_secret` is not exposed in Github logs.
+
+## Inputs
+
+### `base64_toml_content`
+
+- **Description**: Base64-encoded TOML configurations separated by spaces that contain "_secret" keys to be masked.
+- **Required**: Yes
+
+## How to Use
+
+To use this action in your workflow, follow the steps below:
+
+1. Ensure you have a Base64-encoded TOML configuration string that you wish to mask. The keys for any secrets in the TOML should end with `_secret`.
+2. Add a step in your GitHub Actions workflow to use this action. Do it **before** any other action related to the TOML 
```

Comment:
maybe give more examples? 
* if that toml is user input, when should this action run?
* if that toml is generated in the workflow, when should this action run?

Reply:
@Tofel 

> if that toml is user input, when should this action run?

the action should run before you use TOML values. This is in the readme. Please say how would you want to update this text :) 

> if that toml is generated in the workflow, when should this action run?

This action does not care how you generate base64-encoded toml. You just have to pass it to the action

Reply:
I meant: add it to the readme!

Reply:
I want to see specific examples for these 2 cases, readme is too abstract for me, I wouldn't be sure how to use that acitons in these 2 cases